components/prism-core.js

```diff
@@ -417,6 +424,7 @@ script = script[script.length - 1];
 
 if (script) {
 	_.filename = script.src;
+	_.tags = (script.getAttribute('data-tags') || 'code').split(/\s*,\s*/g);
```

Comment:
If they're just tag names why comma separate them? Why not just space-separated?